### ISSUE TITLE
Broken Links?

### ISSUE THREAD

**Issue by jtma67** [department-of-veterans-affairs/va-data#15]
When I tried to access any of the links under Featured Data, what was returned was the source code for the web page.  This was from an iPad 4th gen tablet running the latest iOS.

**Comment by raymondlangley** [comment]
Do you have a moderator?  This would be a useful resource if the site worked as intended.  Please fix the links that are broken there are several

**Comment by Hampshire100** [comment]
Could you please post a screen shot of what your seeing? I have tested the links and they are resolving to their proper .json files. 

1. (https://raw.githubusercontent.com/department-of-veterans-affairs/VHA-Facilities/master/VAFacilityLocation.json)
2. 
3. https://raw.githubusercontent.com/va-data/va-resources/master/homeless-assistance.json
4. 
5. https://raw.githubusercontent.com/va-data/va-resources/master/family-caregiver-support.json

**Comment by QJF3** [comment]
Agree.  I just tried reviewing the above links on iOS 10.3.2 and I can view the source JSON file.  That's how the pages are currently designed, the link is the JSON data file.

https://raw.githubusercontent.com/department-of-veterans-affairs/VHA-Facilities/master/VAFacilityLocation.json

**Comment by raymondlangley** [comment]
I clicked on one of the links and this is what I see

Raymond
